### PR TITLE
FHAC-626: xsd changes

### PR DESCRIPTION
**Pull request by TabassumJafri**
Added EventTimestamp, EventID, EventType, EventOutcomeIndicator, RemoteHCID, RequestMessageId, RelatesTo, TransactionId, UserId and removed interface element from LogEventSecureRequestTypeand LogEventRequestType.

### CODE REVIEW

File: src/main/resources/schemas/nhinc/common/AuditLog.xsd [hunk 1]
```diff
@@ -1,175 +1,188 @@
...
+    <!-- begin generic log message -->
+    <xsd:complexType name="LogEventRequestType">
+        <xsd:sequence>
+            <xsd:element ref="auditmessage:AuditMessage"/>
+            <xsd:element ref="auditlog:direction"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+            <xsd:element name="EventTimestamp" minOccurs="0" type="xsd:dateTime"/>
+            <xsd:element name="EventID" minOccurs="0" type="xsd:string"/>
+            <xsd:element name="EventType" minOccurs="0" type="xsd:string"/>
+            <xsd:element name="EventOutcomeIndicator" minOccurs="0" type="xsd:integer"/>
```
Comment (christophermay07) [inline src/main/resources/schemas/nhinc/common/AuditLog.xsd]: The above four Event* elements are NOT NULL in the database definition; should this be reflected here?

File: src/main/resources/schemas/nhinc/common/AuditLog.xsd [hunk 2]
```diff
@@ -1,175 +1,188 @@
...
+            <xsd:element name="RemoteHCID" minOccurs="0" type="xsd:string"/>
```
Comment (christophermay07) [inline src/main/resources/schemas/nhinc/common/AuditLog.xsd]: This is in line with the database schema, but, can RemoteHCID ever be null?

Comment (TabassumJafri) [inline src/main/resources/schemas/nhinc/common/AuditLog.xsd]: For responding gateway, RemoteHCID could be null

File: src/main/resources/schemas/nhinc/common/AuditLog.xsd (hunk 1 above)
Comment (TabassumJafri) [inline src/main/resources/schemas/nhinc/common/AuditLog.xsd]: Yes they can be defined as maxOccurs=1 so each element will be required only once. (minOccurs=1 is by default)

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Pull request merged**